form/src/main/java/com/ritense/form/service/impl/FormIoFormDefinitionService.java

```diff
@@ -64,6 +64,9 @@ public Optional<FormIoFormDefinition> getFormDefinitionByNameIgnoringCase(String
     @Override
     @Transactional
     public FormIoFormDefinition createFormDefinition(CreateFormDefinitionRequest request) {
+        if(formDefinitionRepository.findByName(request.getName()).isPresent()) {
```

Comment:
So you mean there is no check in the db ? If there is a unique constraint you could catch that error and just report that.

Reply:
There wasn't, I have added a unique constraint on the column now. However, because the error message is rather ambiguous, I think my custom exception is better. This is the message you'll see without the check `org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "name"`